src/plugins/data_view_field_editor/public/components/field_format_editor/editors/color/color.tsx

```diff
@@ -156,6 +163,28 @@ export class ColorFormatEditor extends DefaultFormatEditor<ColorFormatEditorForm
                   item.index
                 );
               }}
+              button={
+                <EuiButton
+                  minWidth="false"
+                  iconType="lettering"
+                  color="text"
+                  onClick={() => {}}
+                  aria-label={`Select a text color for item ${item.index}`}
+                >
+                  <EuiIcon
+                    aria-label={color}
+                    color={color}
+                    size="l"
+                    type="stopFilled"
+                    css={{
+                      stroke: 'var(--euiColorFullShade)',
+                      strokeWidth: 2,
```

Comment:
What do you think about extracting reusable CSS out of component body? It seems like the same in lines 226-229 😀

Reply:
Or maybe we can even extract the whole EuiIcon, as those have 1:1 same props (if we intent to keep it this way of course)

Reply:
i'll remove the CSS. it wasn't working correctly, good catch. I'll also pass the requirement for `EuiColorPickerSwatch` to get a `button` prop to the EUI team in a separate issue so I can use it in this case....sticking the icon into a button is a workaround for this UX and ideally i'd use the swatch (but can't because this is a button in a button). 

sound good?

Reply:
Sounds good to me! 🙌

Reply:
thank you! resolving